apps/meteor/app/api/server/v1/rooms.js

```diff
@@ -333,14 +333,15 @@ API.v1.addRoute(
 		get() {
 			const { offset, count } = this.getPaginationItems();
 			const { sort } = this.parseJsonQuery();
-			const { types, filter } = this.requestParams();
+			const { types, filter, useFname } = this.requestParams();
```

Comment:
This here is something I have some questions about.

Should we expect this value to come from the request, or should we get it directly from the settings? (even if we expect it to come from the request, I think we should get the setting and overwrite the value if it is sent through a parameter)